### PR TITLE
STREAM-3081 Introduce Stream Connect docs

### PR DESCRIPTION
**Pull request by Everlag**
I've dropped the 'hidden' from the index page.

-----

This is the initial pass at Stream Connect docs.

### CODE REVIEW

File: products/stream/src/content/stream-connect/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+  "errors": [],
+  "messages": []
+}
+```
+
+### Start streaming
+
+Use the url and streamKey returned from input creation in your streaming software.
+
+TODO(zaid): obs example screenshots for a step-by-step?
```
Comment (renandincer) [inline products/stream/src/content/stream-connect/getting-started.md]:
```suggestion

```

**Comment by Everlag** [comment]
Force push catches some commits I missed, fork workflow a bit unusual for my usual flow...

File: products/stream/src/content/stream-connect/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+These limits can be configured. Please reply to your beta invite email for more information.
+
+### Get details on a live input
+
+#### cURL example
+
+```bash
+curl -H "Authorization: Bearer $TOKEN" \
+https://api.cloudflare.com/client/v4/accounts/$ACCOUNT/stream/live_inputs/$INPUT_UID
```
Comment (renandincer) [inline products/stream/src/content/stream-connect/getting-started.md:110]:
```suggestion
https://api.cloudflare.com/client/v4/accounts/$ACCOUNT/stream/live_inputs/$INPUT_UID --data '{"meta":{"name":"test stream 1"}}'
```

File: products/stream/src/content/stream-connect/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+Use the URL and streamKey returned from input creation in your streaming software.
+
+TODO(zaid): obs example screenshots for a step-by-step?
+
+## Limits
+
+Some limits apply to the Stream Connect Beta:
+
+- Up to 1000 live inputs per-account can be created
+- Up to 10 outputs may be configured per live input
```
Comment (renandincer) [inline products/stream/src/content/stream-connect/getting-started.md]:
```suggestion
- Up to 50 outputs may be configured per live input
```

**Review by renandincer — APPROVED** [review]

**Pull request merged**